src/Language/PureScript/Bundle.hs

```diff
@@ -411,26 +422,20 @@ compile modules entryPoints = filteredModules
     where
     -- | Create a set of vertices for a module element.
     --
-    -- Some special cases worth commenting on:
-    --
-    -- 1) Regular exports which simply export their own name do not count as dependencies.
-    --    Regular exports which rename and reexport an operator do count, however.
```

Comment:
Is it safe to get rid of the handling of operators like this? Also are there any tests exercising this case?

Reply:
I'll try to cook up a test. The change here though is not to the handling of operators, but to the handling of everything else—now all exports count as dependencies, whether they export their own name or not. I hypothesize that the previous behavior was only needed because there was no distinction between `Public` and `Internal` symbols; now that there is, every export is a `Public` symbol and if that export depends on an `Internal` symbol, regardless of any difference in the names of those symbols, the dependency is tracked.

Reply:
Actually, given that explanation I'm not sure a test of re-exporting operators with different names is necessary, but I'll add it if you still think it is.

Reply:
Thanks, yes that makes sense. I’ll have a think and dig through the existing tests and get back to you.